e2e/gometa/main.go

```diff
@@ -52,6 +53,42 @@ var tests = []test{
 		},
 		expected: "https://fallback-to.gometa.path.example.com",
 	},
+	{
+		name: "Fallback to path record's \"to=\" when path is empty",
+		args: data{
+			host: "custom-regex.gometa.path.example.com",
+			path: "/",
+		},
+		expected: "https://no-path.gometa.path.example.com/",
+		comment:  "for: k8s.io",
+	},
+	{
+		name: "Redirect to gometa record using the custom regex in path record",
+		args: data{
+			host: "custom-regex.gometa.path.example.com",
+			path: "/txtdirect/txtdirect?go-get=1",
+		},
+		expected: "https://package.gometa.path.example.com/txtdirect",
+		comment:  "for: k8s.io",
+	},
+	{
+		name: "Redirect to gometa record using the custom regex and single path",
+		args: data{
+			host: "custom-regex.gometa.path.example.com",
+			path: "/txtdirect?go-get=1",
+		},
+		expected: "https://package.gometa.path.example.com/txtdirect",
+		comment:  "for: k8s.io",
+	},
+	// {
+	// 	name: "Redirect to host record if request doesn't have ?go-get=1",
```

Comment:
As you mentioned in your next comment, it's commented because we can't use negative look ahead. Should we remove it or keep it with a explanation comment about negative look ahead issue?

Reply:
Remove it and open a separate issue. We have git for keeping around historic code and issues for things to keep in mind and work on I think